hack/verify-all.sh

```diff
@@ -117,6 +121,11 @@ if [[ "${res}" = 0 ]]; then
   echo "All verify checks passed, congrats!"
 else
   echo ""
```

Comment:
can this be removed?

Reply:
this is for aligning the output. It's nicer with it in.

Reply:
👍